docs/examples/datasets/pipelines.py

```diff
@@ -350,3 +350,91 @@ def test_multiple_pipelines(client):
     )
     assert df.shape == (4, 4)
     assert found.sum() == 4
+
+
+# docsnip aggregate_with_str_window
```

Comment:
Is some place in documentation using this new docsnip?

Reply:
yup found many in pipelines so copies the syntax, should I remove it?

Reply:
Actually no, we can remove this and change all the existing ones to reflect the change, what say you?

Reply:
Yes, that sounds like the right thing to do here. Thanks!